tests/e2e/actor_activation/actor_activation_test.go

```diff
@@ -78,12 +79,13 @@ func TestMain(m *testing.M) {
 	// and will be cleaned up after all tests are finished automatically
 	testApps := []kube.AppDescription{
 		{
-			AppName:        appName,
-			DaprEnabled:    true,
-			ImageName:      "e2e-actorapp",
-			Replicas:       1,
-			IngressEnabled: true,
-			MetricsEnabled: true,
+			AppName:         appName,
```

Comment:
Existing tests should not require a change - it means it will break existing users.

Reply:
Done.